src/flask/cli.py

```diff
@@ -760,6 +757,11 @@ def _validate_key(ctx, param, value):
     return value
 
 
+def _isAdHoc(cert):
+    """Check is it adhoc"""
```

Comment:
This docstring is the type of documentation that nobody really needs :) It's like "loop over list" above a loop...

Reply:
Yea, Its my problem from go, start write everywhere camecase, can fx this